### ISSUE TITLE
Case-sensitive search for categories

### ISSUE THREAD

**Issue by VeschiyOleh** [moneymanagerex/android-money-manager-ex#1660]
After updating to the April version, search by category became case-sensitive.  This is very inconvenient, given that all categories begin with a capital letter.

**Comment by guanlisheng** [comment]
thanks @VeschiyOleh

what's your previous version?

**Comment by VeschiyOleh** [comment]
2019.02.14. 994-release

**Comment by guanlisheng** [comment]
i c. 
related with #789

**Comment by guanlisheng** [comment]
hi @VeschiyOleh , can you share some categories' names in Cyrillic?

**Comment by VeschiyOleh** [comment]
[your_data.zip](https://github.com/moneymanagerex/android-money-manager-ex/files/14999203/your_data.zip)
I created a test database with Cyrillic categories.

**Comment by guanlisheng** [comment]
thanks @VeschiyOleh , can you show some detailed steps to reproduce the issue, e.g. search which charters?

**Comment by guanlisheng** [comment]
hi @VeschiyOleh this has been addressed since v2024.12.25 https://play.google.com/apps/testing/com.money.manager.ex.android

**Comment by bballmaster** [comment]
Hi @guanlisheng,

This issue was fixed but re-appeared right in the next release - https://github.com/moneymanagerex/android-money-manager-ex/releases/tag/2024.12.27.1056 - and is present since then.

**Comment by wolfsolver** [comment]
Are you sure that category is not inactive? take a look to Entity - Category and check if your category active. is not long press and activate.

Taking a look to you sample DB new category was inactive

**Comment by bballmaster** [comment]
Yes, I am sure that the category is active. The same problem with case-sensitive filter concerns the "Payer" field and the search field in reports as well.

![Image](https://github.com/user-attachments/assets/c19b357a-571d-4857-a40e-5046a264ae5c)
![Image](https://github.com/user-attachments/assets/fc4f8d92-cae7-4cc5-bbe3-45d8c52f03d3)

**Comment by wolfsolver** [comment]
I try with  dB previous message https://github.com/moneymanagerex/android-money-manager-ex/issues/1660#issuecomment-2059376900

Without issue.

Can you give me a sample dB and a couple of search strings?

Are you able to check log to see if there is some useful info?

**Comment by bballmaster** [comment]
Hello again @wolfsolver.

Sorry for the delay. Here is a sample DB with some Cyrillic Categories, Payers and Accounts: [data_cyrillic.zip](https://github.com/user-attachments/files/19120421/data_cyrillic.zip).

BTW I checked the DB from the comment https://github.com/moneymanagerex/android-money-manager-ex/issues/1660#issuecomment-2059376900 - and the behavior is the same: search and filtering finds Cyrillic options with matching case for symbols only.
